src/core/src/sys/cache.rs

```diff
@@ -162,21 +162,6 @@ impl EnergyCache {
             }
         }
 
-        // Interactions withing the sub-system being moved
-        for (i, &part_i) in idxes.iter().enumerate() {
-            for (j, &part_j) in idxes.iter().enumerate().skip(i + 1) {
-                let r = system.cell().distance(&newpos[i], &newpos[j]);
-                let energy = evaluator.pair(r, part_i, part_j);
-
-                pairs_delta += energy;
-                new_pairs[(part_i, part_j)] += energy;
-                new_pairs[(part_j, part_i)] += energy;
-
-                pairs_delta -= self.pairs_cache[(part_i, part_j)];
-            }
-        }
-
```

Comment:
This may be reintroduced in the next step of this PR. Currently, I can think of no scenario where the intra and intermolecular structure of multiple (but not all) particles/molecules may change. If all change, like in volume changes, we should write an own caching function.